app-modules/engagement/src/Notifications/EngagementFailedNotification.php

```diff
@@ -72,7 +72,7 @@ public function toDatabase(object $notifiable): array
         return FilamentNotification::make()
             ->danger()
             ->title('Engagement Delivery Failed')
-            ->body("Your engagement failed to be delivered to {$this->engagement->recipient->display_name}.")
+            ->body("Your engagement email failed to be delivered to {$this->engagement->recipient->display_name}.")
```

Comment:
As discussed, let's make sure to say `sms` or `email` here depending on the type of the notification that failed.

Reply:
@Orrison Made the changes.